applications/mne_analyze/plugins/annotationmanager/annotationsettingsview.cpp

```diff
@@ -329,3 +369,49 @@ void AnnotationSettingsView::realTimeDataTime(double dValue)
     m_pAnnModel->updateFilteredSample(t_iSample);
     this->onDataChanged();
 }
+
+//=============================================================================================================
+
+void AnnotationSettingsView::newUserGroup(QString sName, int iType)
```

Comment:
Use const reference `const QString& sName` and add linebreak before second parameter